stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -128,7 +139,6 @@ class AssignManualQuantsLines(models.TransientModel):
     assign_wizard = fields.Many2one(
         comodel_name="assign.manual.quants",
         string="Move",
-        required=True,
```

Comment:
we don't need here `required` as it forces an error when `_onchange_selected` is played
the field `assign_wizard` will be anyway propagated with the value, because one2many creation is handled

Reply:
@jcadhoc I've checked different implementations of one2many lines for a wizard.
The code is definitely ok.
IDK why there is an issue with the `assign_wizard` field being checked for required when onchange `selected` called.
It should be checked on creation.

Reply:
@yankinmax I tested the version merged in 16.0 and got the same error:
https://github.com/OCA/stock-logistics-warehouse/issues/1736

Reply:
@BT-jmichaud I've tried functional testing several times on real database and demo database. Locally I don't have such issue. On the runbot it exists. I have no idea why. If you know where the problem comes from please open a PR to fix that.
The way one2many field on the wizard is filled with data is ok.